tiledb/sm/c_api/tiledb.h

```diff
@@ -1215,6 +1215,10 @@ TILEDB_EXPORT void tiledb_config_free(tiledb_config_t** config);
  * - `rest.ignore_ssl_validation` <br>
  *    Have curl ignore ssl peer and host validation for REST server. <br>
  *    **Default**: false
+ * - `rest.creation_access_credentials_name` <br>
+ *    The name of the registered access key to use for creation of the REST
+ *    server. <br>
+ *    **Default**: ""
```

Comment:
This does not actually have a default value. I'm not sure what the proper way to document this is. In my experience, I have only ever added new configuration parameters with a default value.

Reply:
I switched to say `no default set`. What do you think of that?

Reply:
Fine by me, thanks!